src/pages/Shopping/Shopping.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import ShoppingItem from './components/ShoppingItem/ShoppingItem';
+import './Shopping.scss';
+
+class Shopping extends Component {
+  constructor() {
+    super();
+    this.state = {
+      product: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/shoppingData.json')
+      .then(response => response.json())
+      .then(product => {
+        this.setState({
+          product,
+        });
+      });
+  }
+
+  render() {
+    const { product } = this.state;
+
+    return (
+      <main className="shopping">
+        <div className="grid-container">
+          <div className="row">
+            <div className="col-lg-1 lg-only"></div>
+            <div className="col-sm-4 col-md-8 col-lg-8 ">
+              <article>
+                <header>
+                  <h1>장바구니</h1>
+                  <button>...</button>
+                </header>
+                <div>
+                  {product.map(p => {
+                    return <ShoppingItem key={p.id} product={p} />;
+                  })}
```

Comment:
p라는게 product의 줄임말인가요? 적합하지 않아보입니다!
```suggestion
                  {productList.map(product => {
                    return <ShoppingItem key={product.id} product={product} />;
                  })}
```
변수명 일부 수정해서 이런식으로 하시는건 어떨까요?
선경님 PR에서도 비슷하게 활용하신 것 보였는데 혹시 같이 작업하신건가요?

Reply:
```suggestion
                  {product.map(p => <ShoppingItem key={p.id} product={p} />;
                  )}
```
arrow function에서 함수 본문에서 바로 return 할 경우 위와 같이 중괄호와 return keyword 생략할 수 있습니다!